### ISSUE TITLE
Social Media

### ISSUE THREAD

**Issue by zoombear** [ghaida/gdisjc-website#8]
- map!
- Show GDI twitter feed
- Tell your friends about GDI
- Facebook
- Linkedin
- Twitter
- Google+

**Comment by zoombear** [comment]
I assigned you because I thought you'd enjoy doing this part :) Feel free to un-assign yourself or sign up for another issue if you'd like!

**Comment by zoombear** [comment]
I put a section on the wiki to share sketches, ideas...etc
[Wiki](https://github.com/ghaida/gdisjc-website/wiki/Social-Media-Map)

**Comment by arc6789** [comment]
I am interested :) Should I close this issue then? Or does that happen once the development is done?

**Comment by zoombear** [comment]
Nope, don't close it yet! Let's wait till it's done :)
